support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -109,36 +118,37 @@ object SendAcquisitionEvent {
         }
       }
 
-      override def buildAcquisition(state: SendAcquisitionEventState): Either[String, thrift.Acquisition] = {
-        val (productType, productAmount) = state.product match {
+      override def buildAcquisition(a: SendAcquisitionEventStateAndRequestInfo): Either[String, thrift.Acquisition] = {
```

Comment:
maybe this parameter could be named stateAndInfo or something more descriptive than 'a'

Reply:
done!